src/widgets/vitals/vitals-card.resource.tsx

```diff
@@ -97,16 +115,18 @@ function formatVitals(
       diastolic: diastolic && diastolic.valueQuantity.value,
       pulse: pulse && pulse.valueQuantity.value,
       temperature: temperature && temperature.valueQuantity.value,
-      oxygenation: oxygenation && oxygenation.valueQuantity.value
+      oxygenation: oxygenation && oxygenation.valueQuantity.value,
+      weight: weight && weight.valueQuantity.value,
+      height: height && height.valueQuantity.value
     });
   });
 }
 
-function getDatesIssued(vitalsArray): string[] {
-  return vitalsArray.map(vitals => vitals.issued);
+function getDatesIssued(vitalsArray): Array<Date> {
+  return vitalsArray?.map(vitals => vitals.issued);
 }
 
-function latestFirst(a, b) {
+function latestFirst(a: Date, b: Date) {
   return new Date(b).getTime() - new Date(a).getTime();
```

Comment:
It is cast to Date but not instantiated, therefore at runtime it fails

Reply:
Well, but then the provided typing is wrong.

My recommendation would be to change it to whatever it is in reality, e.g., `string` or `number`. Otherwise, instantiate beforehand.